Assets/Scripts/OrchestraArmy/Entity/Entities/Projectiles/Controllers/FinalBossProjectileMovementController.cs

```diff
@@ -48,18 +48,7 @@ public void HandleMovement()
                     Entity = Entity
                 };
             }
-            
-            // If the distance is far enough, we can move on to the homing projectiles.
-            if (Vector3.Distance(StartingPosition, Entity.transform.position) >= DistanceBeforeSeek)
-            {
-                MovementController = new PlayerFollowingProjectileMovementController()
-                {
-                    Entity = Entity,
-                    Player = Player
-                };
-            }
-            
-            
```

Comment:
Waarom is dit weggehaald? Is een requirement voor The Conductor

Reply:
Kan het wel terug zetten, maar 3-5 projectielen van verschillende kanten die je ook nog volgen ga je nooit ontwijken xD

Reply:
teruggezet, maar t is nu wel brutal. Orchestra Army: prepare to die edition